### PR TITLE
Feature/141 aggregations overview

### PR DESCRIPTION
**Pull request by nikkite99**

### CODE REVIEW

File: backend/src/main/java/de/amos/apachepulsarui/service/ClusterService.java [hunk 1]
```diff
@@ -85,4 +89,20 @@ private List<String> getTenantsAllowedForCluster(String clusterName) throws Puls
...
+        List<String> tenats = getTenantsAllowedForCluster(clusterDto.getName());
+        long numberOfNamespaces = tenats.stream().mapToLong(this::getNumberOfNamespacesForTenant).sum();
+        clusterDto.setNumberOfTenants(tenats.size());
+        clusterDto.setNumberOfNamespces(numberOfNamespaces);
+        return clusterDto;
+    }
+
+    private long getNumberOfNamespacesForTenant(String tenant) {
+        try {
+            return pulsarAdmin.namespaces().getNamespaces(tenant).size();
```
Comment (jnsrnhld) [inline backend/src/main/java/de/amos/apachepulsarui/service/ClusterService.java:102]: Can't we use the method from the namespace service?
`namespaceService.getAllOfTenant(tenant).size()`

File: backend/src/main/java/de/amos/apachepulsarui/service/NamespaceService.java [hunk 2]
```diff
@@ -80,7 +80,13 @@ private NamespaceDetailDto enrichWithNamespaceData(NamespaceDetailDto namespace)
     }
 
     private NamespaceDto enrichWithCardDetails(NamespaceDto namespace) {
-            //TODO fill with information when given by pos
+        try {
+            namespace.setNumberOfTopics(pulsarAdmin.namespaces().getTopics(namespace.getId()).size());
```
Comment (jnsrnhld) [inline backend/src/main/java/de/amos/apachepulsarui/service/NamespaceService.java:84]: Same here. Can't we use the topic service for this?
`topicService.getAllByNamespace(topic).size();`

File: backend/src/main/java/de/amos/apachepulsarui/service/TenantService.java [hunk 3]
```diff
@@ -71,4 +72,26 @@ private TenantInfo getTenantInfo(String tenantName) {
...
+            tenantDto.setNumberOfNamespaces(namespaces.size());
+            return tenantDto;
+        } catch (PulsarAdminException e) {
+            throw new PulsarApiException(
+                    "Could not fetch tenant data of tenant '%s'".formatted(tenantDto.getName()), e
+            );
+        }
+    }
+
+    private List<String> getNumberOfTopicsForNamespace(String namespace) {
```
Comment (jnsrnhld) [inline backend/src/main/java/de/amos/apachepulsarui/service/TenantService.java:90]: I guess this is just an orphan method? Seems duplicated.

**Review by ahaverkamp — APPROVED** [review]

**Review by jnsrnhld — APPROVED** [review]
lgtm

**Pull request merged**